resources/idlharness.js

```diff
@@ -2160,7 +2176,17 @@ IdlInterface.prototype.test_immutable_prototype = function(type, obj)
             } catch (err) {}
         });
 
-        assert_throws(new TypeError(), function() {
+        var setter;
```

Comment:
This block is complex enough that I think it deserves a quick comment as to why it needs to exist.

Reply:
Yep, will add.